### PR TITLE
Update Gradle, remove Vision files

### PR DESCRIPTION
**Pull request by kiryldz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,11 @@
...
-Some of devices that will work with VisionSDK:
-- Samsung Galaxy S8, S8+ // S9, S9+ // Note 8
-- Xiaomi Mi 6 // 8
-- HTC U11, U11+ // U12, U12+
-- OnePlus 5 // 6
-
-You can also check more details at [Vision SDK FAQ](https://vision.mapbox.com/#faq).
+For more information and frequently asked questions please visit [Vision SDK FAQ](https://vision.mapbox.com/#faq).
 
 ## Installation and setup
```
Comment (yunikkk) [inline README.md]: I'd also alter text here since Examples is the only module remained

File: README.md [hunk 2]
```diff
@@ -1,26 +1,11 @@
...
-VisionSDK requires Android 6 (API 23) and higher, with QC Snapdragon 650 // 710 // 8xx with Open CL support
-
-Some of devices that will work with VisionSDK:
-- Samsung Galaxy S8, S8+ // S9, S9+ // Note 8
-- Xiaomi Mi 6 // 8
-- HTC U11, U11+ // U12, U12+
-- OnePlus 5 // 6
-
-You can also check more details at [Vision SDK FAQ](https://vision.mapbox.com/#faq).
+For more information and frequently asked questions please visit [Vision SDK FAQ](https://vision.mapbox.com/#faq).
```
Comment (yunikkk) [inline README.md]: Not sure this faq is worth mentioning since it's pretty outdated. Docs links at the bottom seem enough to me

Comment (kiryldz) [reply README.md]: Removing that block then.

File: README.md (hunk 1 above)
Comment (kiryldz) [reply README.md]: OK.

**Pull request merged**